SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -139,7 +139,14 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	APawn* GetLocalFlowPawn();
 
 	template <class T>
-	T* SpawnActor(const FActorSpawnParameters& SpawnParameters = FActorSpawnParameters(), const bool bRegisterAsAutoDestroy = true);
+	T* SpawnActor(const bool bRegisterAsAutoDestroy);
+
+	template <class T>
+	T* SpawnActor(const FActorSpawnParameters& SpawnParameters);
```

Comment:
why no bRegister?

Reply:
see one of the resolved convos w/ simon, was decided one method with everything defaulted and then 2 others with common use cases would be good